activities/migrations/0015_auto_20171109_0905.py

```diff
@@ -0,0 +1,41 @@
+# -*- coding: utf-8 -*-
```

Comment:
Make one migration file for all the changes in the database at a time

Reply:
ok